fontforgeexe/charview.c

```diff
@@ -3933,7 +3933,7 @@ static void CVCharUp(CharView *cv, GEvent *event ) {
 	    TRACE("was on charselector\n");
 	    GWidgetIndicateFocusGadget( cv->hsb );
 	}
-	else
+	else if ( cv->charselector != NULL )
```

Comment:
@frank-trampe it's explained in the [commit message](https://github.com/jtanx/fontforge/commit/074fe7586fc50dd3b04288fd2ab11dc02e2f9464).